evm/operators.js

```diff
@@ -229,6 +242,49 @@ async function processCommand(_, chain, options) {
             break;
         }
 
+        case 'updateGasInfo': {
+            const target = chain.contracts.AxelarGasService?.address;
+
+            validateParameters({
+                isNonEmptyStringArray: { chains },
+                isAddress: { target },
+            });
+
+            let gasUpdates = await getGasUpdates(config, env, chain, chains);
+
+            gasUpdates = gasUpdates.filter((update) => update !== null);
+
+            // Adding lowercase chain names for case insensitivity
+            gasUpdates.forEach(({ chainName, gasInfo }) => {
+                if (chainName.toLowerCase() !== chainName) {
+                    gasUpdates.push({
+                        chainName: chainName.toLowerCase(),
+                        gasInfo,
+                    });
+                }
+            });
+
+            const filteredChains = gasUpdates.map(({ chainName }) => chainName);
+            const gasInfoUpdates = gasUpdates.map(({ gasInfo }) => gasInfo);
```

Comment:
move this into the shared util? it can return the chains and gasInfo to be used for the update